docs/outlook/onmessagefromchanged-onappointmentfromchanged-events.md

```diff
@@ -55,8 +55,94 @@ Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeo
 
 ## Configure the manifest
 
-> [!NOTE]
-> The `OnMessageFromChanged` and `OnAppointmentFromChanged` events aren't yet supported for the [Unified manifest for Microsoft 365 (preview)](../develop/unified-manifest-overview.md).
+# [Unified manifest for Microsoft 365 (developer preview)](#tab/jsonmanifest)
+
+1. Open the **manifest.json** file.
+
+1. Add the following object to the "extensions.runtimes" array. Note the following about this markup.
+
+   - The "minVersion" of the Mailbox requirement set is configured as "1.13" because this is the lowest version of the requirement set that supports the `OnMessageFromChanged` event. For more information, see the "Supported events" table in [Configure your Outlook add-in for event-based activation](autolaunch.md#supported-events).
+   - The "id" of the runtime is set to a descriptive name, "autorun_runtime".
+   - The "code" property has a child "page" property set to an HTML file and a child "script" property set to a JavaScript file. You'll create or edit these files in later steps. Office uses one of these values depending on the platform.
+       - Outlook on Windows executes the event handler in a JavaScript-only runtime, which loads a JavaScript file directly.
+       - Outlook on Mac and on the web execute the handler in a browser runtime, which loads an HTML file. The HTML file contains a `<script>` tag that then loads the JavaScript file.
+
+     For more information, see [Runtimes in Office Add-ins](../testing/runtimes.md).
+   - The "lifetime" property is set to "short". This means the runtime starts up when the event occurs and shuts down when the handler completes.
+   - There are "actions" to run handlers for the `OnMessageFromChanged` and `OnNewMessageCompose` events. You'll create the handlers in a later step.
+
+    ```json
+    {
+        "requirements": {
+            "capabilities": [
+                {
+                    "name": "Mailbox",
+                    "minVersion": "1.13"
+                }
+            ]
+        },
+        "id": "autorun_runtime",
+        "type": "general",
+        "code": {
+            "page": "https://localhost:3000/commands.html",
+            "script": "https://localhost:3000/launchevent.js"
+        },
+        "lifetime": "short",
+        "actions": [
+            {
+                "id": "onMessageFromChangedHandler",
+                "type": "executeFunction",
+                "displayName": "onMessageFromChangedHandler"
+            },
+            {
+                "id": "onNewMessageComposeHandler",
+                "type": "executeFunction",
+                "displayName": "onNewMessageComposeHandler"
+            }
+        ]
+    }
+    ```
+
+1. Add the following "autoRunEvents" array as a property of the object in the "extensions" array.
+
+    ```json
+    "autoRunEvents": [
+    
+    ]
+    ```
+
+1. Add the following object to the "autoRunEvents" array.
```

Comment:
Why spit this into two steps?

Reply:
Thanks for pointing this out. This step was based on previous manifest guides, which made sense to split up at the time. However, for this walkthrough, we can definitely merge the two steps.